### DISCUSSION TITLE
HOLD tap or Layer tap?

### DISCUSSION THREAD

**Discussion by RopNopSLop** [nickcoutsos/keymap-editor#211] [General]
Hi im trying to set some "hold" home row mods, where on tap/kp is just the letter.. but when held it activates the layer until the HOLD is released .... is this HOLD/TAP or layer/TAP?

I came from zsa moonlander, so something like this 
<img width="453" alt="holdtap or layertap?or new behavior?" src="https://github.com/nickcoutsos/keymap-editor/assets/166423976/9fc5b1fe-f295-4e6f-aefb-efaa2051ff8d">

**Comment by nickcoutsos** [comment]
"Layer tap" is a specific implementation of the generic "hold tap" behavior. "Mod tap" is another variation that accepts two keycodes so it can be used for homerow mods like `&mt LCMD A`. ZMK's [hold-tap documentation](https://zmk.dev/docs/behaviors/hold-tap#example-use-cases) shows how you might create a custom behavior to further tailor it to homerow mods. You can either set that up in the keymap code and load it into the editor or create it within the app under the behaviors tab.
